arbnode/node.go

```diff
@@ -250,11 +302,19 @@ func DeployOnL1(ctx context.Context, parentChainReader *headerreader.HeaderReade
 		validatorAddrs = append(validatorAddrs, crypto.CreateAddress(validatorWalletCreator, i))
 	}
 
+	deployParams := rollupgen.RollupCreatorRollupDeploymentParams{
+		Config:                    config,
+		BatchPoster:               batchPoster,
+		Validators:                validatorAddrs,
+		MaxDataSize:               maxDataSize,
+		NativeToken:               nativeToken,
+		DeployFactoriesToL2:       false,
+		MaxFeePerGasForRetryables: big.NewInt(0), // needed when utility factories are deployed
```

Comment:
can we have those deployed?

Reply:
added here - https://github.com/OffchainLabs/nitro/pull/1879/commits/1e5a97b2139a6df1793af90143b180c326929ef8

Reply:
as agreed we keep it as false for now, and we'll later add support in testnode to switch it to true

Reply:
yup, it seems to clutter nitro go deployment setup (and messing up some testcase) if we default to deploy those on L2; maybe we can add a flag in the testnode script to deploy those afterward like the token bridge, but for now I think we should simply do it within hardhat if we need them